src/main/target/AIKONF4/target.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+
+#include <platform.h>
+#include "drivers/io.h"
+
+#include "drivers/dma.h"
+#include "drivers/timer.h"
+#include "drivers/timer_def.h"
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+    DEF_TIM(TIM2,  CH2,  PB3,  TIM_USE_ANY,                 0, 0), // CAMERA_CONTROL_PIN
+
+    // TIM8_UP (DMA2_ST1_CH7)
+    DEF_TIM(TIM8,  CH1,  PC6,  TIM_USE_MOTOR,               0, 0), // M1
+    DEF_TIM(TIM8,  CH2,  PC7,  TIM_USE_MOTOR,               0, 0), // M2
+    DEF_TIM(TIM8,  CH3,  PC8,  TIM_USE_MOTOR,               0, 0), // M3
+    DEF_TIM(TIM8,  CH4,  PC9,  TIM_USE_MOTOR,               0, 0), // M4
+    // TIM3_UP (DMA1_ST2_CH5)
+    DEF_TIM(TIM3,  CH3,  PB0,  TIM_USE_MOTOR,               0, 0), // M5
+    DEF_TIM(TIM3,  CH4,  PB1,  TIM_USE_MOTOR,               0, 0), // M6
+
+    // backdoor timers
+    DEF_TIM(TIM1,  CH2,  PA9,  TIM_USE_ANY,                 0, 0), // UART1_TX
```

Comment:
Does it make sense to define backdoor timers on pins that are connected to an inverter? What can these pins be used for (except for being used by the UART controlling the inverter)?

Reply:
Indeed doesn't, removed it.